source/simulator/helper_functions.cc

```diff
@@ -1484,6 +1484,212 @@ namespace aspect
 
 
   template <int dim>
+  void Simulator<dim>::compute_reactions ()
+  {
+    if (time_step == 0)
+      return;
```

Comment:
can you add why this works? (I suspect it's because the first time step length is zero, right? that could be checked by an assertion.)